ledger/simulation/simulator_test.go

```diff
@@ -108,7 +109,7 @@ func TestSimulateWithTrace(t *testing.T) {
 
 	env := simulationtesting.PrepareSimulatorTest(t)
 	defer env.Close()
-	s := MakeSimulator(env.Ledger)
+	s := MakeSimulator(env.Ledger, config.GetDefaultLocal().EnableDeveloperAPI)
```

Comment:
Shouldn't this just pass in `true`?  You want to confirm that traces appear when asked for, not that the default config allows them, right?

Reply:
my bad for confusing you. I believe this test is not quite related with this PR, i.e., exposing PC trace.

Though the name suggests `Trace`, but it is not related with the execution trace here. It should follow the default `EnableDeveloperAPI`, which is false.